tutorials/asr/06_Voice_Activiy_Detection.ipynb

```diff
@@ -0,0 +1,1151 @@
+    "\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "\"\"\"\n",
+    "# If you're using Google Colab and not running locally, run this cell.\n",
+    "\n",
+    "## Install dependencies\n",
+    "!pip install wget\n",
+    "!apt-get install sox libsndfile1 ffmpeg\n",
+    "!pip install unidecode\n",
+    "\n",
+    "# ## Install NeMo\n",
+    "!python -m pip install --upgrade git+https://github.com/NVIDIA/NeMo.git@candidate#egg=nemo_toolkit[asr]\n",
+    "\n",
+    "## Install TorchAudio\n",
+    "!pip install torchaudio>=0.6.0 -f https://download.pytorch.org/whl/torch_stable.html\n",
+    "\n",
+    "## Grab the config we'll use in this example\n",
+    "!mkdir configs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "This VAD tutorial is based on the MatchboxNet model from the paper \"[MatchboxNet: 1D Time-Channel Separable Convolutional Neural Network Architecture for Speech Commands Recognition](https://arxiv.org/abs/2004.08531)\" with a modified decoder head to suit classification tasks.\n",
+    "\n",
+    "The notebook will follow the steps below:\n",
+    "\n",
+    " - Dataset preparation: Instruction of downloading datasets. And how to convert it to a format suitable for use with nemo_asr\n",
+    " - Audio preprocessing (feature extraction): signal normalization, windowing, (log) spectrogram (or mel scale spectrogram, or MFCC)\n",
+    "\n",
+    " - Data augmentation using SpecAugment \"[SpecAugment: A Simple Data Augmentation Method for Automatic Speech Recognition](https://arxiv.org/abs/1904.08779)\" to increase number of data samples.\n",
+    " \n",
+    " - Develop a small Neural classification model which can be trained efficiently.\n",
+    " \n",
+    " - Model training on the Google Speech Commands dataset and Freesound dataset in NeMo.\n",
+    " \n",
+    " - Evaluation of error cases of the model by audibly hearing the samples\n",
+    " \n",
+    " - Add more evaluation metrics and transfer learning/fine tune\n",
+    " "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "I62_LJzc-p2b"
+   },
+   "outputs": [],
+   "source": [
+    "# Some utility imports\n",
+    "import os\n",
+    "from omegaconf import OmegaConf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "K_M8wpkwd7d7"
+   },
+   "source": [
+    "# Data Preparation\n",
```

Comment:
this cell can't be run. Should this be text rather than code cell?

Reply:
emm it indeed is text locally but turns out to 'changed' to code cell in colab... trouble shooting it.

Reply:
fixed!